webpack.config.js

```diff
@@ -13,6 +13,7 @@ const config = {
 		blocks: './blocks/index.js',
 		editor: './editor/index.js',
 		element: './element/index.js',
+		keycodes: './keycodes/index.js',
```

Comment:
Do we really want a separate module for `keycodes`? This raises the question of whether we'd need a generic module: "utils", "common" or something like that.